apps/api/src/app/health/health.controller.ts

```diff
@@ -1,22 +1,19 @@
 import { Controller, Get } from '@nestjs/common';
-import { ApiExcludeController } from '@nestjs/swagger';
-import {
-  HealthCheck,
-  HealthCheckResult,
-  HealthCheckService,
-  HealthIndicatorFunction,
-  HealthCheckError,
-} from '@nestjs/terminus';
+import { ApiBearerAuth, ApiExcludeController } from '@nestjs/swagger';
+import { HealthCheck, HealthCheckResult, HealthCheckService, HealthIndicatorFunction } from '@nestjs/terminus';
 import {
   CacheServiceHealthIndicator,
   DalServiceHealthIndicator,
   WorkflowQueueServiceHealthIndicator,
 } from '@novu/application-generic';
 
 import { version } from '../../../package.json';
+import { ApiKeyAuth } from '../shared/framework/swagger/api.key.security';
 
 @Controller('health-check')
 @ApiExcludeController()
+@ApiBearerAuth()
+@ApiKeyAuth()
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** on this controller I think that we don't need these annotations